csrc/executor.h

```diff
@@ -31,6 +31,7 @@ struct TORCH_CUDA_CU_API CompileOptions {
 class TORCH_CUDA_CU_API FusionExecutor : public NonCopyable {
  public:
   struct GlobalBufferInfo {
+    TensorView* tv = nullptr;
```

Comment:
IIUC, we are using TensorView in `transformOutputFromAllocationToRFactor` just to get the correct stride/format on output tensors? Does that mean the `strides` here is just useless?

Reply:
I think the `strides` would be still useful if you want the executor to allocate discontiguous tensor. If you only want to support user provided discontiguous tensor, then strides won't be needed.